AsyncDisplayKit/Details/UIView+ASConvenience.h

```diff
@@ -42,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)setNeedsDisplay;
 - (void)setNeedsLayout;
```

Comment:
My thought was that we could double-declare setNeedsLayout in .h and describe it next to its partner invalidateSize.  However let's not do this now -- later diff.  

Probably remove layoutIfNeeded for now due to the comments in the property bridge impl of that method.